File: src/app/landingPage/models/EditTeam.ts
import {EditTeamMember} from './EditTeamMember'

export class EditTeam {
    public constructor(
        public id: string,
        public name: string,
        public description: string,
        public teamLeaderId: string,
        public teamManagerId: string,
    public teamMembers: EditTeamMember[]) { }
}

File: ViewModels/EditTeam.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace HonestProject.ViewModels
{
    public class EditTeam
    {
        public Guid Id {get; set;}
        public string Name { get; set; }
        public string Description { get; set; }
        public EditTeamMember[] TeamMembers { get; set; }
        public Guid TeamLeaderId { get; set; }
        public Guid TeamManagerId { get; set; }
    }
}